cli/install.go

```diff
@@ -400,6 +429,38 @@ func (z *ZVM) InstallZls(version string) error {
 	return nil
 }
 
+func findZlsExecutable(dir string) (string, error) {
+	var result string
+
+	filename := "zls"
+	if runtime.GOOS == "windows" {
+		filename += ".exe"
+	}
+
+	err := filepath.Walk(dir, filepath.WalkFunc(func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		// Discard directories and symlinks (if any)
+		if info.IsDir() || info.Mode().Type() == os.ModeSymlink {
+			return nil
+		}
+
+		if filepath.Base(path) != filename {
+			return nil
+		}
+		result = path
```

Comment:
Can't verify if ZLS candidate is executable in a portable way since Windows doesn't mark files as executable